rspec-rails.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.homepage    = "https://github.com/rspec/rspec-rails"
   s.summary     = "RSpec for Rails"
   s.description = "rspec-rails is a testing framework for Rails 5+."
-  s.required_ruby_version = ">= 2.2.0"
+  s.required_ruby_version = ">= 2.7.0"
```

Comment:
[Rails 6.1 is 2.5+](https://rubygems.org/gems/rails/versions/6.1.0). Do we want to make it stricter?

Reply:
ALL THE STRICT

Reply:
Probably should be 2.5 then, sad times :p